images/installer/Dockerfile

```
@@ -10,7 +10,7 @@ COPY images/installer/origin-extra-root /
 # install ansible and deps
 RUN INSTALL_PKGS="python-lxml python-dns pyOpenSSL python2-cryptography openssl python2-passlib httpd-tools openssh-clients origin-clients iproute patch" \
  && yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS \
- && EPEL_PKGS="ansible-2.7.4 python2-boto python2-crypto which python2-pip.noarch python2-scandir python2-packaging azure-cli-2.0.46" \
+ && EPEL_PKGS="ansible-2.7.7 python2-boto python2-crypto which python2-pip.noarch python2-scandir python2-packaging azure-cli-2.0.46" \
```

```suggestion
 && EPEL_PKGS="ansible-2.7 python2-boto python2-crypto which python2-pip.noarch python2-scandir python2-packaging azure-cli-2.0.46" \
```
Lets be less specific about the version, as the repo is now pointing to a specific minor version

---

I agree, let's generalize this so we aren't constantly in a race against ci jobs. Looks like all of the jobs are failing now due to 2.7.8 being the only available version in the repos.

---

Oh, yum can't do that - it either needs `ansible` (which means it would pick one from epel) or exact version string, so it has to be `ansible-2.7.8` currently

---

I was looking at 2.7.8 in epel, specifically in the dockerfile. Are we installing from the openshift repos normally?

---

Pushed that change to unblock CI

---

I've tested `ansible-2.7*`.  It will get any 2.7 version.  This was all just an intermediate step to get to 2.8.  I know all this stuff comes out of the woodworks when we start making changes so I wanted to find the pain points early.  I would have gone to 2.7.8 directly but when I opened this PR, 2.7.8 was not released to Fedora yet.